pkg/core/system.go

```diff
@@ -21,18 +21,22 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/projectriff/riff/pkg/fileutils"
 	"os"
 	"strings"
 	"time"
 
-	"github.com/projectriff/riff/pkg/fileutils"
-
 	"github.com/projectriff/riff/pkg/env"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const istioNamespace = "istio-system"
+const (
+	istioNamespace = "istio-system"
+
+	retryDuration = 5 * time.Second
+	retrySteps    = 60
```

Comment:
isn't this a bit excessive, I've never had a case where I had to try more than 3-4 times. So if there is a real issue other than the time it takes to create the cluster resources we would just be spinning our wheels for 5 minutes.

Reply:
I'm happy to reduce `retrySteps` to 5 on that basis.